builder/virtualbox/ovf/config_test.go

```diff
@@ -30,24 +30,6 @@ func getTempFile(t *testing.T) *os.File {
 	return tf
 }
 
-func testConfigErr(t *testing.T, warns []string, err error) {
```

Comment:
hmm good point. go actually recommends against assertion functions. https://golang.org/doc/faq#testing_framework

probably ok to leave it like this. I guess in the future the better way to do it would be to have the assertion functions return errors like


```
func testOK(warns []string, err error) error {
        if len(warns) > 0 {
 		return str.Errorf("bad: %#v", warns)
 	}
 	if err != nil {
 		return str.Errorf("bad: %s", err)
 	}
 }

if err := testOK(...) ; err != nil {
         t.Fatalf(err)
}
```